cs/v1/CS_v1.0.0.yaml

```diff
@@ -874,6 +960,303 @@ components:
       required:
         - carrierServiceName
         - vesselSchedules
+    CutOffTime:
+      title: CutOffTime
+      x-stoplight:
+        id: 0s6yu6v21mabq
+      type: object
+      description: Cut Off Times.
+      required:
+        - cutOffDateTimeCode
+        - cutOffDateTime
+      properties:
+        cutOffDateTimeCode:
+          type: string
+          x-stoplight:
+            id: 6kxo2zw6m656u
+          description: "Code for the cut-off time. Possible values are:\r\n- `DCO` (Documentation cut-off)\r\n- `VCO` (VGM cut-off)\r\n- `FCO` (FCL delivery cut-off)\r\n- `LCO` (LCL delivery cut-off)\r\n- `PCO` (Port cut-off)\r\n- `ECP` (Empty container pick-up date and time)\r\n- `EFC` (Earliest full-container delivery date)\r\n- `RCO` (Reefer cut-off)\r\n- `DGC` (Dangerous Goods cut-off)\r\n- `OBC` (OOG/BB cut-off)\r\n- `TCO` (Transhipment cut-off)\r\n- `STA` (Standard booking acceptance)\r\n- `SPA` (Special booking acceptance)\r\n- `CUA` (Customs Acceptance)\r\n- `AFC` (Advanced filling cut-off)\r\n\r\n**LCL Condition** : Only applicable when the `Receipt Type at Origin` is `CFS`.\r\n\r\nMore details can be found on [GitHub](https://github.com/dcsaorg/DCSA-OpenAPI/blob/master/reference-data/CutOffDateTimeCodeList-CS_v1.0.0.csv)"
+          maxLength: 3
+          example: DCO
+        cutOffDateTime:
+          type: string
+          x-stoplight:
+            id: ovjrw1tb3wbdx
+          format: date-time
+          description: Actual cut-off time.
+          example: '2019-11-12T07:41:00-08:30'
+    AddressLocation:
+      title: AddressLocation
+      x-stoplight:
+        id: m8siaz93if4cf
+      type: object
+      description: An interface used to express a location using an `Address` object.
+      required:
+        - locationType
+        - address
+      properties:
+        locationName:
+          type: string
+          x-stoplight:
+            id: 1qntskkr8v0h5
+          example: Port of Amsterdam
+          description: The name of the location.
+          maxLength: 100
+        locationType:
+          type: string
+          maxLength: 4
+          description: Discriminator used to identify this as an `Address` location interface.
+          example: ADDR
+        address:
+          type: object
+          description: An object for storing address related information.
+          required:
+            - name
+          properties:
+            name:
```

Comment:
This doesn't follow DT - in DT we have removed name... (Argument from Flavia was that we have locationName to cover for this